### PR TITLE
Fix iOS browser crash on GPU Accelerated img limit

### PR DESCRIPTION
**Pull request by jam35L**
Use translate instead of translate3d.

### CODE REVIEW

**Comment by xiaolin** [comment]
Thanks for the PR! I will look at this when I get back next week!

File: src/ImageGallery.jsx [hunk 1]
```diff
@@ -719,32 +719,32 @@ export default class ImageGallery extends React.Component {
       translateX = this._getTranslateXForTwoSlide(index);
     }
 
-    const translate3d = `translate3d(${translateX}%, 0, 0)`;
+    const = `translate(${translateX}%, 0)`;
```
Comment (xiaolin) [inline src/ImageGallery.jsx]: forgot the variable here `translate`

**Comment by xiaolin** [comment]
@jam35L if you can fix the issue, I can pull down and test. Thanks!

**Comment by jam35L** [comment]
Requested changes is done.

**Comment by xiaolin** [comment]
Cool @jam35L been pretttty busy, but will test this out soon and merge.

**Comment by xiaolin** [comment]
Merged by rebase and squashing

**Pull request closed without merging**